src/main/java/org/sagebionetworks/bridge/models/surveys/SurveyElementSQL.java

```diff
@@ -0,0 +1,50 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.List;
+
+import org.sagebionetworks.bridge.hibernate.HibernateSurvey;
+import org.sagebionetworks.bridge.hibernate.HibernateSurveyElement;
+import org.sagebionetworks.bridge.json.JsonNodeToSurveyElementConverter;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+@JsonDeserialize(as=HibernateSurveyElement.class)
+public interface SurveyElementSQL extends BridgeEntity {
```

Comment:
This class looks identical to SurveyElement, except before/afterRules is a JsonNode instead of SurveyRules. Is there any reason why this needs to be a separate interface? Why not have HibernateSurveyElement and DynamoSurveyElement both share the same interface?